src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,183 @@
+
+Get a container registry
+-------------
+::
+
+    Command
+        az acr show: Get a container registry.
+
+    Arguments
+        --name -n [Required]: Name of container registry.
+        --resource-group -g : Name of resource group.
+
+Update a container registry
+-------------
+::
+
+    Command
+        az acr update: Update a container registry.
+
+    Arguments
+        --name -n [Required]: Name of container registry.
+        --app-id            : The app id of an existing service principal. If provided, no --new-sp or
+                            -p should be specified.
+        --disable-admin     : Disable admin user.
+        --enable-admin      : Enable admin user.
+        --new-sp            : Create a new service principal. If provided, no --app-id should be
+                            specified. Optional: Use -p to specify a password.
+        --password -p       : Password used to log into a container registry.
+        --resource-group -g : Name of resource group.
+        --role -r           : Name of role. Allowed roles: owner, contributor, reader.  Default: reader.
+        --tags              : Space separated tags in 'key[=value]' format. Use "" to clear existing
+                            tags.
+        --tenant-id -t      : Tenant id for service principal login. Warning: Changing tenant id will
+                            invalidate assigned access of existing service principals.
+
+    Examples
+        Update tags of a container registry
+            az acr update -n myRegistry --tags key1=value1;key2=value2
+        Update a container registry with a new service principal
+            az acr update -n myRegistry --new-sp -p myPassword -r Owner
+        Update a container registry with an existing service principal
+            az acr update -n myRegistry --app-id myAppId -r Owner
+
+Update storage account for a container registry
+-------------
+::
+
+    Command
+        az acr storage update: Update storage account for a container registry.
+
+    Arguments
+        --name -n                 [Required]: Name of container registry.
+        --storage-account-name -s [Required]: Name of new or existing storage account.
+        --resource-group -g                 : Name of resource group.
+
+Get admin username and password for a container registry
+-------------
+::
+
+    Command
+        az acr credential show: Get admin username and password for a container registry.
+
+    Arguments
+        --name -n [Required]: Name of container registry.
+        --resource-group -g : Name of resource group.
+
+List repositories in a given container registry
+-------------
+::
+
+    Command
+        az acr repository list: List repositories in a given container registry.
+
+    Arguments
+        --name -n [Required]: Name of container registry.
+        --password -p       : Password used to log into a container registry.
+        --username -u       : Username used to log into a container registry.
```

Comment:
Added interactive username/password when they are not provided or cannot be retrieved.